File: Numbers.py
#Ask for numbers to display the next ten numbers, squares, square roots, and cubes
#By Aisha Jackson
#10/18/14

#!/usr/bin/python

import math #import math function

print "Hello user! This program is going to show some things relative to the number you choose! \n"

num = int(input("Please enter any integer. \n")) #ask for the number

count = 1
while count < 11: #get the number, square, cube and square root
    num = num + 1
    num2 = num ** 2
    num3 = num ** 3
    num4 = math.sqrt (num)
    print "NUMBER: " + str(num) + " SQUARE: " + str(num2) + " CUBE: " + str(num3) + " SQUARE ROOT: " + str(num4)
    count = count + 1

File: Numbers.java
//Program that asks for numbers to display the next ten numbers, squares, square roots, and cubes
//Written by Aisha Jackson
// Sept 26, 2014

import java.util.Scanner; //import the Scanner class

public class Numbers { //class declaration
    
    public static void main(String []args) //program execution starts
    {
        Scanner keyboard = new Scanner(System.in); //single shared Scanner for keyboard input

        System.out.println("Hello user! This program is going to show some things relative to the number you choose!");
        
        System.out.println("Please enter any integer.");
        int num = keyboard.nextInt();

        for(int i =1; i < 11; i++){
            num++;
            double num2 = Math.pow(num, 2);
            double num3 = Math.pow(num, 3);
            double num4 = Math.sqrt(num);
            System.out.println("NUMBER: " + num + " SQUARE: "+ num2+ " CUBE: "+ num3+ " SQUARE ROOT: "+ num4);
        }
    }
}